examples/cloud-operations/network-dashboard/cloud-function/main.py

```diff
@@ -0,0 +1,791 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud import monitoring_v3
+from googleapiclient import discovery
+from google.api import metric_pb2 as ga_metric
+import time
+import os
+import google.api_core
+import re
+import random
+
+# 
+monitored_projects_list = os.environ.get("monitored_projects_list").split(",")  # list of projects from which function will get quotas information
```

Comment:
These are constants, they should be all uppercase. Can you also move inline comments ton their own line, and make sure you observe line length limits? In Fabric we standardize on yapf for formatting, you should use it as it will trigger PR check errors if you don't.

Reply:
Done, thanks.